kubernetes/bankapp-ingress.yml

```
@@ -3,26 +3,29 @@ kind: Ingress
-  - host: megaproject.trainwithshubham.com
+  - host: megaproject.sandeeppokhrel.cloud
     http:
-      paths:
-      - path: /
-        pathType: Prefix
-        backend:
+       paths:
+       - path: /
+         pathType: Prefix
+         backend:
           service:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 11 but found 10

(indentation)